std/algorithm/comparison.d

```diff
@@ -1565,3 +1568,228 @@ unittest
                 2, "two",
                 ));
 }
+
+/**
+Returns true if all of the elements in $(D r1) are present in $(D r2). Otherwise,
+returns false.
```

Comment:
Not quite a correct description. I think the conditions are that r1 and r2 have the same number of elements, and elements appear in r1 the same number of times as they do in r2.